src/less-style/structure.less

```diff
@@ -37,6 +37,23 @@ body.vellum-full-screen {
 
                 }
 
+                .fd-commander {
+                    display: flex;
+                    padding: 6px 4px 6px 12px;
+                    background-color: @cc-dark-cool-accent-mid;
+
+                    .fd-add-question-toggle {
+                        color: white;
+                        height: 17px;
+                    }
+
+                    input {
+                        border: 1px solid @cc-dark-cool-accent-low;
+                        flex: 2;
+                        margin: -2px 0;
```

Comment:
I like the idea of having an X button at the right end of the commander to dismiss it. I had originally wanted to keep the plus (+) icon to the left for that very purpose, especially for non-power users who accidentally get into the command bar and then don't know how to add a question. An X button is more intuitive for dismissing the commander. Great idea!

Don't really like the idea of showing the commander below the add question button, since that will either result in a content shift (jarring in its own way) or obscure part of the tree (quite possibly the part that the user wants to interact with), both of which seem undesirable. When you say "above add question" do you mean it would overlay the form name and menu button? That might be a reasonable place for it once you can invoke all of the menu items with the commander... (not yet)

I made it overlay/replace the add question button because I think that button will probably not be used very often by people who use the commander, and certainly not _while_ using the commander. Does it seem like a reasonable compromise if there is an X on the right to dismiss it and show the add question button?